tests/clusters/test_cluster.py

```diff
@@ -196,3 +219,15 @@ def test_cache_connections() -> None:
     assert cluster_1.get_query_connection(
         cluster.ClickhouseClientSettings.QUERY
     ) != cluster_3.get_query_connection(cluster.ClickhouseClientSettings.QUERY)
+
+
+@patch("snuba.settings.PARTITIONED_CLUSTERS", PARTITIONED_CLUSTERS_CONFIG)
+def test_partitioned_cluster() -> None:
```

Comment:
can we also test that `get_cluster` on partition 0 returns the other cluster values? (it can be in the same method)

Reply:
Done